packages/catalog-search/src/SearchPagination.jsx

```diff
@@ -1,7 +1,7 @@
-import React, { useContext, useMemo } from 'react';
+import React, { useContext } from 'react';
```

Comment:
[nit/non-blocking] Do we still need to import `React`. If not is it a good opportunity to stop importing it if we don't need it.

Reply:
Good suggestion, but if I change it here, we should just change it everywhere. I also recall during development where I did try to remove it in a few places while working on the PR, but it complained (in tests) that `React` couldn't be found without it imported. Will defer for separate contribution/fix.